.github/workflows/build.yaml

```diff
@@ -407,4 +407,43 @@ jobs:
         with:
           sourceDirectory: /home/runner/tmp/third-party-dependencies
           distribution: xenial
-          component: ${{ env.GITHUB_REF }}
\ No newline at end of file
+          component: ${{ env.GITHUB_REF }}
+
+  build-n-publish-PyPI:
+    name: Build and Publish Python Packages to PyPI
+    runs-on: ubuntu-20.04
+    needs: [workflow-setup, indy_plenum_tests, indy_plenum_module_tests, lint]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Python 3.5
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
+
+      - name: Install required packages via pip
+        run: | 
+          python3 -m pip install pytest-runner wheel pip==9.0.3 
+
+      - name: Set Build Version
```

Comment:
# Could be removed here and upstreamVer could be used from the job build_release. However, setting the build version here as well allows for parallel processing